src/AccessibilityInsights.SharedUx/Controls/HierarchyControl.xaml

```diff
@@ -136,19 +136,19 @@
                             <MenuItem x:Name="mniRaw" IsCheckable="False" Click="mniRaw_Click"
                                   AutomationProperties.Name="{x:Static Properties:Resources.mniRawAutomationPropertiesName1}">
                                 <MenuItem.Header>
-                                    <RadioButton x:Name="rbRaw" Content="_Raw" Loaded="mniRaw_Loaded" Click="rbRaw_Click" Focusable="False" />
+                                    <RadioButton x:Name="rbRaw" Content="{x:Static Properties:Resources.HierarchyControl_Raw}" Loaded="mniRaw_Loaded" Click="rbRaw_Click" Focusable="False" />
                                 </MenuItem.Header>
                             </MenuItem>
                             <MenuItem x:Name="mniControl" IsCheckable="False" Click="mniControl_Click" 
                                   AutomationProperties.Name="{x:Static Properties:Resources.mniControlAutomationPropertiesName}">
                                 <MenuItem.Header>
-                                    <RadioButton x:Name="rbControl" Content="_Control" Loaded="mniControl_Loaded" Click="rbControl_Click" Focusable="False" />
+                                    <RadioButton x:Name="rbControl" Content="{x:Static Properties:Resources.HierarchyControl_Control}" Loaded="mniControl_Loaded" Click="rbControl_Click" Focusable="False" />
                                 </MenuItem.Header>
                             </MenuItem>
                             <MenuItem x:Name="mniContent" IsCheckable="False" Click="mniContent_Click"
                                   AutomationProperties.Name="{x:Static Properties:Resources.mniContentAutomationPropertiesName1}">
                                 <MenuItem.Header>
-                                    <RadioButton x:Name="rbContent" Content="Con_tent" Loaded="mniContent_Loaded" Click="rbContent_Click" Focusable="False" />
+                                    <RadioButton x:Name="rbContent" Content="{x:Static Properties:Resources.HierarchyControl_Con_tent}" Loaded="mniContent_Loaded" Click="rbContent_Click" Focusable="False" />
```

Comment:
The underscore on these strings indicates the keyboard accelerator key within the menu, so _Raw is rendered with an underscored R, _Control is rendered with an underscored C, and Content is rendered with an underscored t. Since we aren't using the underscores in the names Raw and Control, we probably shouldn't use it for Content, either. That would make the name here `HierarchyControl_Content`.